examples/ping-pong/src/main.rs

```diff
@@ -69,6 +69,7 @@ async fn main() {
     // GraphcastAgentConfig defines the configuration that the SDK expects from all Radios, regardless of their specific functionality
```

Comment:
The two `.clone()`s added in this file are not necessary, and the one that's already there can be removed too :wink: